server/gokbg3/grails-app/services/org/gokb/LanguagesService.groovy

```diff
@@ -0,0 +1,43 @@
+package org.gokb
+
+import grails.gorm.transactions.Transactional
+import grails.util.Holders
+import groovyx.net.http.HTTPBuilder
+import groovyx.net.http.RESTClient
+import org.gokb.cred.KBComponent
+import org.gokb.cred.RefdataCategory
+
+import static groovyx.net.http.Method.GET
+
+@Transactional
+class LanguagesService{
+
+  static Map localesToLanguages = [
+      "DE" : "ger",
+      "EN" : "eng",
+      "FR" : "fre"
+  ]
+
+  static Map languages = [:]
+
+  
+  static void initialize(){
+    String uriString = "${Holders.grailsApplication.config.gokb.languagesUrl}/api/listIso639two"
```

Comment:
The whole url including path should probably be the config value. Would be great to have a link to an API specification in a comment here as well.

Reply:
The idea is that the config file keeps the central languages API URL, whereas the single endpoints are addressed in the code. Because, unlike this initial version, further API functionality might become used. E. g. in order to show a full language-specific name of a language, you might use the endpoint `api/language` instead of holding all language names inside the (this) application. There might be further use language API use cases. You can include all endpoints in the config file, if this really your choice. The thing is just that keeping several endpoints for one microservice per `environment` crowds your config. But it's a matter of taste I would say.

Reply:
I see your point. In that case it would be really helpful to refer to a detailed spec for each endpoint though.

Reply:
Added documentation in G3Readme.md, the wiki and on README.md of the languages microservice.